scripts/eurostat/health_determinants/common/denominator_mcf_generator.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+import os
+import sys
+
+_COMMON_PATH = os.path.abspath(os.path.dirname(__file__))
+sys.path.insert(1, _COMMON_PATH)
+from dcid_existance import check_dcid_existance
+
+
+def _generate_mcf(sv_list, mcf_file_path) -> None:
```

Comment:
This function's parsing and logic is so fragile that it would be nice to have a unit test just dedicated to testing this. I recommend adding a couple of unit tests for it.